substrate/frame/broker/src/benchmarking.rs

```diff
@@ -771,7 +771,7 @@ mod benches {
 		let core_count = n.try_into().unwrap();
 		let config = new_config_record::<T>();
 
-		let now = frame_system::Pallet::<T>::block_number();
+		let now = <<<T as crate::Config>::Coretime as CoretimeInterface>::RealyChainBlockNumberProvider as BlockNumberProvider>::current_block_number();
```

Comment:
This typo has been fixed on master. You can merge master and fix `RealyChainBlockNumberProvider` -> `RelayChainBlockNumberProvider`